lib/mongo/socket/ssl.rb

```diff
@@ -114,16 +114,51 @@ def connectable?
 
       private
 
+      def verify_certificate?
+        @verify_certificate ||=
+          # If ssl_verify_certificate is not present, disable only if ssl_verify is
+          # explicitly set to false.
+          if options[:ssl_verify_certificate].nil?
+            options[:ssl_verify] != false
+          # If ssl_verify_certificate is present, enable or disable based on its value.
+          else
+            options[:ssl_verify_certificate]
```

Comment:
Can this be spelled `!!options[:ssl_verify_certificate]` to convert it to a boolean and possibly a unit test added to verify precedence?

Reply:
Sure